openmetadata-ui/src/main/resources/ui/src/components/Assets/AssetsSelectionModal/AssetSelectionModal.tsx

```diff
@@ -80,22 +83,37 @@ export const AssetSelectionModal = ({
   const [totalCount, setTotalCount] = useState(0);
 
   const [isSaveLoading, setIsSaveLoading] = useState<boolean>(false);
+  const [aggregations, setAggregations] = useState<Aggregations>();
+  const [selectedQuickFilters, setSelectedQuickFilters] = useState<
+    ExploreQuickFilterField[]
+  >([] as ExploreQuickFilterField[]);
+  const [quickFilterQuery, setQuickFilterQuery] = useState<
+    QueryFilterInterface | undefined
```

Comment:
```suggestion
    QueryFilterInterface
```
if we dont provide value at initial state by default it will be `QueryFilterInterface | undefined`